Ratan

RatanShreshtha opened the pull request:
Added Readme.md

RatanShreshtha committed:
First Commit

RatanShreshtha committed:
Some changes made

RatanShreshtha committed:
Readme.md added

RatanShreshtha committed:
Update Readme.md

RatanShreshtha committed:
Update Readme.md

shubhamgoyal commented:
@RatanShreshtha I managed to get the server up and running because I know Django, but your Readme is not very useful for a new developer. You didn't handle the case where the roles or databases didn't exist.

Anyways, I have no idea how to use your application. Could you create a proper user guide? I don't want to go back and forth many times, so please try to make sure that I have everything I need this time :)

Kshitij9210 commented:
@RatanShreshtha : Also try to ensure that all commands are properly vetted before any commits. Eg. in the Readme.md we have "python manage.py migerate". Appreciate if you could come up with a proper documentation :)

Thanks,
Kshitij

RatanShreshtha committed:
Charting added.

RatanShreshtha committed:
New Model to handle large symbol.

RatanShreshtha committed:
Portfolio edit option added.

RatanShreshtha committed:
Portfolio edit option fixed.

RatanShreshtha committed:
Instructions and Readme.Md added and modified.

RatanShreshtha committed:
Instructions and Readme.Md added and modified.

RatanShreshtha committed:
Readme.Md modified.

RatanShreshtha committed:
Readme.Md fixed.

RatanShreshtha commented:
As you have instructed I have made a concise documentation and added the case when user does not exists in the Readme.md.

And added synopsis of the project.
